doc/api/v8.md

```diff
@@ -22,6 +22,7 @@ Returns an object with the following properties:
 * `total_available_size` {number}
 * `used_heap_size` {number}
 * `heap_size_limit` {number}
+* `does_zap_garbage` {bool}
```

Comment:
I don't think it's all that useful without explaining why it matters if garbage is getting zapped or not.

`--zap_code_space` makes V8 overwrite heap garbage with a bit pattern.  The RSS footprint (resident memory set) gets bigger because it continuously touches all heap pages and that makes them less likely to get swapped out by the operating system.